src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/Dialogs/Queries/Get/GetDialogDto.cs

```diff
@@ -12,183 +12,644 @@ namespace Digdir.Domain.Dialogporten.Application.Features.V1.EndUser.Dialogs.Que
+
+    /// <summary>
+    /// Boolean indicating if the endpoint is deprecated. Integrators should migrate to endpoints with a higher version.
+    /// </summary>
     public bool Deprecated { get; set; }
+
+    /// <summary>
+    /// Date and time when the service owner has indicated that endpoint will no longer function. Only set if the endpoint
+    /// is deprecated. Dialogporten will not enforce this date.
+    /// </summary>
     public DateTimeOffset? SunsetAt { get; set; }
 }
 
 public sealed class GetDialogDialogGuiActionDto
 {
+    /// <summary>
+    /// The unique identifier for the action in UUIDv7 format.
+    /// </summary>
     public Guid Id { get; set; }
+
+    /// <summary>
+    /// The action identifier for the action, corresponding to the "action" attributeId used in the XACML service policy,
+    /// </summary>
     public string Action { get; set; } = null!;
+
+    /// <summary>
+    /// The fully qualified HTTPS URL of the action, to which the user will be redirected when the action is triggered. Will be set to
+    /// "urn:dialogporten:unauthorized" if the user is not authorized to perform the action.
+    /// </summary>
+    /// <example>
+    /// urn:dialogporten:unauthorized
+    /// https://someendpoint.com/gui/some-service-instance-id
+    /// </example>
     public Uri Url { get; set; } = null!;
+
+    /// <summary>
+    /// Contains an authorization resource attributeId, that can used in custom authorization rules in the XACML service
+    /// policy, which by default is the policy belonging to the service referred to by "serviceResource" in the dialog.
+    ///
+    /// Can also be used to refer to other service policies.
+    /// </summary>
+    /// <example>
+    /// mycustomresource
+    /// /* equivalent to the above */
+    /// urn:altinn:subresource:mycustomresource
+    /// urn:altinn:task:Task_1
+    /// /* refer to another service */
+    /// urn:altinn:resource:some-other-service-identifier
+    /// </example>
     public string? AuthorizationAttribute { get; set; }
+
+    /// <summary>
+    /// Whether the user is authorized to perform the action.
+    /// </summary>
     public bool IsAuthorized { get; set; }
+
+    /// <summary>
+    /// Indicates whether the action results in the dialog being deleted. Used by frontends to implement custom UX
+    /// for delete actions.
+    /// </summary>
     public bool IsDeleteDialogAction { get; set; }
 
+    /// <summary>
+    /// Indicates a priority for the action, making it possible for frontends to adapt GUI elements based on action
+    /// priority.
+    /// </summary>
     public DialogGuiActionPriority.Values Priority { get; set; }
+
+    /// <summary>
+    /// The HTTP method that the frontend should use when redirecting the user.
+    /// </summary>
     public HttpVerb.Values HttpMethod { get; set; }
 
+    /// <summary>
+    /// The title of the action, this should be short and in verb form. Always text/plain.
```

Comment:
Flisespikk? På denne (og kanskje noen andre) lister av localizationdto nevner du "always text/plain", men du er ikke konsekvent. Dersom jeg er en som skal integrere meg med DP kan jeg se på tittel og tenke "ok jeg kan bruke denne uten å være bekymret for å trenge å escape noe som helst", før jeg leser meg videre på `Prompt` hvor det ikke står noe om MediaType og lure på om jeg må håndtere den forskjellig fra `Title`. Dersom vi ikke er konsekvent på om vi skal inkludere/ekskludere media type fra `List<LocalizationDto>` summary tror jeg vi genererer unødvendig forvirring hos konsumentene våre.

Alle lister av localizationdto er text/plain med unntak av de som er i en content med en MediaType som tillater noe annet.

Reply:
Du har rett, dette kan misforstås. Som konsument av sluttbruker-API-et bør jeg bare forholde meg til mediatype, og i utgangspunktet escape alt og evt. vurdere om jeg ønsker å rendre html/markdown hvis noen felter har det. Fjernet "always text/plain" fra alle steder.